app/models/form_attachment.rb

```diff
@@ -34,6 +34,24 @@ def get_file
 
   private
 
+  def unlock_pdf(file, file_password)
+    return file unless File.extname(file) == '.pdf'
+
+    pdftk = PdfForms.new(Settings.binaries.pdftk)
+    tmpf = Tempfile.new('decrypted')
+    error_messages = pdftk.call_pdftk(file.tempfile.path, 'input_pw', file_password, 'output', tmpf.path)
```

Comment:
Also not sure if it matters but do you need to call tmpf.close before it replaces the current temp file?